presto-docs/src/main/sphinx/release/release-0.288.rst

```diff
@@ -0,0 +1,85 @@
+* Add support for querying system.runtime.tasks table in native clusters.
+* Remove deprecated feature and configuration property ``deprecated.group-by-uses-equal``, which allowed group by to use equal to rather than distinct semantics.
+* ... Improve C++ based Presto documentation :pr:`22717`.
+* Change error code for cast from DOUBLE or REAL to BIGINT, INTEGER, SMALLINT or TINYINT for out of range values from ``NUMERIC_VALUE_OUT_OF_RANGE`` to ``INVALID_CAST_ARGUMENT``.
+* Change handling of floating point numbers in Presto to consistently treat NaNs as larger than any other number and equal to itself.  It also changes the handling of positive and negative zero to always be considered equal to each other.  Read more here: https://github.com/prestodb/rfcs/blob/main/RFC-0001-nan-definition.md. The new nan behavior can be disabled by setting the configuration property ``use-new-nan-definition`` to ``false``. This configuration property is intended to be temporary to ease migration in the short term, and will be removed in a future release.
+* Enable HBO for CTE materialized query :pr:`22606`.
+* Prestissimo support for CTAS into bucketed (but not partitioned) tables pr:`22737`.
+* Update CI pipeline to build and publish native worker docker image :pr:`22806`.
+* Upgrade Alluxio to 313.
+* Upgrade io.jsonwebtoken artifacts to 0.11.5 :pr:`22762`.
+* Upgrades fasterxml.jackson artifacts to 2.11 :pr:`22417`.
+
+Hive Connector Changes
+______________________
+* Improve affinity scheduling granularity from a file to a section of a file by adding a `hive.affinity-scheduling-file-section-size` configuration property and `affinity_scheduling_file_section_size` session property. The default file size is 256MB. :pr:`22563`.
+
+Iceberg Connector Changes
+_________________________
+* Improve the partition specs that must be checked to determine if the partition supports metadata deletion or predicate thoroughly pushdown :pr:`22753`.
+* Improve time travel ``TIMESTAMP (SYSTEM_TIME)`` syntax to include timestamp-with-time-zone data type :pr:`22851`.
+* Improve time travel ``VERSION (SYSTEM_VERSION)`` syntax to include snapshot id using bigint data type :pr:`22851`.
+* Add procedure `expire_snapshots` to remove old snapshots in Iceberg. :pr:`22609`.
+* Add support for Iceberg REST catalog :pr:`22417`.
+* Add time travel ``BEFORE`` syntax for Iceberg tables to return historical data :pr:`22851`.
+* Disable timestamp with time zone in create, alter and insert statements :pr:`22926`.
+
+Verifier Changes
+________________
+* Support function call substitution based on the specified substitution pattern passed by the parameter --function-substitutes.
+
+SPI Changes
+___________
+* Add runtime stats as parameter to `ConnectorPageSourceProvider`.
+
+Hive Changes
+____________
+* Introduce AWS Security Mapping which will allow flexible mapping of Presto Users to AWS Credentials or IAM Roles for different AWS Services.
+
```

Comment:
* Fix hash calculation for Timestamp column to be hive compatible when writing to a table bucketed by Timestamp.  :pr:`22890`
* Add config `hive.legacy-timestamp-bucketing` and session property ``hive.legacy_timestamp_bucketing`` to use the original hash function for Timestamp column, which is not hive compatible. :pr:`22890`

Reply:
updated